agent/src/util/constants.rs

```diff
@@ -11,10 +11,10 @@ pub const K8S_DEVICE_PLUGIN_VERSION: &str = "v1beta1";
 
 /// DevicePluginPath is the folder the kubelet expects to find Device-Plugin sockets. Only privileged pods have access to this path.
 #[cfg(not(test))]
-pub const DEVICE_PLUGIN_PATH: &str = "/var/lib/kubelet/device-plugins/";
+pub const DEVICE_PLUGIN_PATH: &str = "/var/lib/kubelet/device-plugins";
 /// Path for testing `DevicePluginService`
 #[cfg(test)]
-pub const DEVICE_PLUGIN_PATH: &str = "/tmp/device-plugins/";
+pub const DEVICE_PLUGIN_PATH: &str = "dummy";
```

Comment:
Is the dummy definition needed?

Reply:
I think so, the product code still builds when I run cargo test. I could conditionally compile the code that uses it away when building test, but that did not sound right. How do you propose to solve this?

Reply:
Just one definition of the variable, no cfg attribute?

Reply:
Good point. Was thinking this will not make folks assume that the value is used for test case. But I think the simplification is better option. Will take out the test value.